prefect_kubernetes/worker.py

```diff
@@ -172,9 +172,15 @@ def _get_default_job_manifest_template() -> Dict[str, Any]:
             "generateName": "{{ name }}-",
         },
         "spec": {
-            "backoffLimit": 0,
+            "backoffLimit": "{{ backoff_limit }}",
             "ttlSecondsAfterFinished": "{{ finished_job_ttl }}",
             "template": {
+                "metadata": {
+                    "annotations": {
+                        "cluster-autoscaler.kubernetes.io/"
+                        "safe-to-evict": "{{ safe_to_evict }}"
```

Comment:
This line separation was completed due to conflicting rules between black and flake8. Reference the docs [here](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#line-length). I am open to alternative options, but the options I saw were: the line break, increasing the line length arbitrarily, or ignoring the E501 rule.

Reply:
Feel free to add a comment to ignore the rule!